src/cdk/a11y/live-announcer/live-announcer.ts

```diff
@@ -149,17 +159,59 @@ export class LiveAnnouncer implements OnDestroy {
     }
   }
 
-  private _createLiveElement(): HTMLElement {
-    const elementClass = 'cdk-live-announcer-element';
-    const previousElements = this._document.getElementsByClassName(elementClass);
-    const liveEl = this._document.createElement('div');
+  private _enqueueAnnouncement(
+      message: string, politeness: AriaLivePoliteness, duration?: number): Promise<void> {
+    const liveElement = this._createLiveElement();
+    // TODO: ensure changing the politeness works on all environments we support.
+    liveElement.setAttribute('aria-live', politeness);
```

Comment:
The politeness should be set at the same time the queued announcement is made, otherwise it will get out of sync. Both the immediate and queued code path should really call the same method that sets the politeness and updates the `textContent`